### PR TITLE
feat: show submitted votes on a voting screen

### LINKED ISSUE #82: you should see your existing vote on a voting screen
**Given** 
Voter submitted an X "Yes" votes for a proposal Y.
Voter clicks on proposal again

**Expected**
Voter should see voting screen with "Yes" selected and vote slider set to X.

**Actual**
Voter sees a "Jetzt Voten" button.

### PR DESCRIPTION
**Pull request by troggy**
Resolves #82, #43, #44, #83, #2, #90, #68 

In progress: #26

Features:
- after the vote, closing the voting control and go to proposal list
- disables "Yes" or "No" button if the vote is already casted
- allows to add/remove votes from existing vote (only "Yes" path tested so far)
- disallow to drag slider to 0 for existing votes

### CODE REVIEW

File: src/volt/components/Header/index.js [hunk 1]
```diff
@@ -16,8 +17,16 @@ import {
   Value
 } from "./styles";
 
+import { factor18 } from '../../utils';
+
+const BN = Web3.utils.BN;
+
 export const Header = props => {
   const { credits, maxCredits = 25, openMenu } = props;
+  let availableCredits = '--';
```
Comment (johba37) [inline src/volt/components/Header/index.js:26]: 🤔

File: src/volt/components/VoteControls/index.js [hunk 2]
```diff
@@ -633,29 +681,44 @@ class VoteControls extends Component {
...
       { value: "yes", color: "voltBrandGreen" },
       { value: "no", color: "voltBrandRed" }
     ];
-    const disabled = votes < 1 || choice === "";
+    console.log({ castedVotes });
+    const castedCredits = castedVotes.mul(castedVotes).div(factor18);
+    console.log({ credits, castedCredits });
+
+    const totalCredits = castedCredits.add(credits || new BN(0)).div(factor18);
+    // no sqrt in BN.js 🤷‍
```
Comment (johba37) [inline src/volt/components/VoteControls/index.js:698]: 🤔

**Review by johba37 — APPROVED** [review]
🥇

File: src/volt/components/Header/index.js (hunk 1 above)
Comment (troggy) [reply src/volt/components/Header/index.js:26]: just copied existing behavior

File: src/volt/components/VoteControls/index.js (hunk 2 above)
Comment (troggy) [reply src/volt/components/VoteControls/index.js:698]: indeed

**Pull request merged**